File: languages/C/fib.c
#include <stdio.h> 
#include <time.h>
int main ()
{
    clock_t t;
    t = clock();
    int fib(int n) {
        if (n <= 1) {
            return n;
        }
        return fib(n-1) + fib(n-2);
    }
    fib(45);
    t = clock() - t;
    double time_taken = ((double)t)/CLOCKS_PER_SEC;
    printf("%f seconds to execute\n", time_taken);
    return 0;
}

File: languages/Rust/fib.rs
use std::time::Instant;
fn fib(n: i32) -> i32 {
    if n <= 1 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}
fn main() {
    let start_time = Instant::now();
    let n = 45;
    fib(n);
    let elapsed_time = start_time.elapsed();
    let time_taken = elapsed_time.as_secs_f64();
    println!("{} seconds to execute", time_taken);
}
